### PR TITLE
feat: add cvss v4 score and vector in scan response

### PR DESCRIPTION
**Pull request by gshanbhag525**
## Description
NVD (NIST) has started publishing cvss v4 for CVE.
eg. CVE-2024-7450,CVE-2024-7451,CVE-2024-7452
Currently trivy doesnot return cvssv4 score, vector in scan response. 
This PR handles it.

## Related issues
- Refer #7059

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: rpc/common/service.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.0
-// 	protoc        v3.19.4
+// 	protoc-gen-go v1.34.2
+// 	protoc        v3.17.3
```
Comment (knqyf263) [inline rpc/common/service.pb.go:4]: How did you generate it? Did you run `mage protoc`?

Comment (gshanbhag525) [reply rpc/common/service.pb.go:4]: I used a local protoc. If needed, I can run  `mage protoc`.
can you please tell me, how it needs to be run.

Comment (knqyf263) [reply rpc/common/service.pb.go:4]: We should use the same version unless we explicitly change the protoc version.

Comment (gshanbhag525) [reply rpc/common/service.pb.go]: fixed , please review.

Comment (knqyf263) [reply rpc/common/service.pb.go]: Thanks

**Review by knqyf263 — APPROVED** [review]
LGTM

**Pull request merged**